baybe/campaign.py

```diff
@@ -58,24 +60,42 @@ class Campaign(SerialMixin):
     strategy: Strategy = field(factory=TwoPhaseStrategy)
     """The employed strategy"""
 
-    # Data
-    measurements_exp: pd.DataFrame = field(factory=pd.DataFrame, eq=eq_dataframe)
-    """The experimental representation of the conducted experiments."""
-
-    numerical_measurements_must_be_within_tolerance: bool = field(default=True)
-    """Flag for forcing numerical measurements to be within tolerance."""
-
     # Metadata
-    n_batches_done: int = field(default=0)
+    n_batches_done: int = field(default=0, init=False)
```

Comment:
Did you build the documentation locally and verify that all of this is rendered properly? Just to make sure :)

Reply:
Jup, looking gucci 💎